src/arrays/creating_arrays/hard/q5/AlphabetRays.java

```diff
@@ -0,0 +1,22 @@
+package arrays.creating_arrays.hard.q5;
+
+/*
+Write a Java program called AlphabetRays to initialize an integer array with all of the lowercase letters of the English
+alphabet and print each character on a separate line. Recall that the character a has the integer equivalent value of 97.
+Sample output:
+a
+b
+…
+y
+z
+ */
+
+public class AlphabetRays {
+    public static void main(String[] args) {
```

Comment:
Actually, we should maybe talk about questions like this because theoretically they could print the letters and not put them into an array.

Reply:
We also do not have a good way of testing if they actually create the array.